xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-docker/src/main/java/org/xwiki/test/docker/junit5/XWikiDockerExtension.java

```diff
@@ -247,7 +249,8 @@ private BrowserWebDriverContainer startBrowser(TestConfiguration testConfigurati
             .withDesiredCapabilities(testConfiguration.getBrowser().getCapabilities())
             .withNetwork(Network.SHARED)
             .withNetworkAliases("vnchost")
-            .withRecordingMode(BrowserWebDriverContainer.VncRecordingMode.SKIP, null);
+            .withRecordingMode(BrowserWebDriverContainer.VncRecordingMode.SKIP, null)
+            .withFileSystemBind(TEST_RESOURCES_HOST_PATH, testConfiguration.getTestResourcesPath());
```

Comment:
This is not going to be performant, but it depends on the volume of data. In any case I'd recommend to do what we do for the servlet container.

Reply:
In any case this is missing some documentation too. See the doc you removed below from the Servlet container for example ;)